undertow/src/main/java/org/wildfly/extension/undertow/UndertowSubsystemSchema.java

```diff
@@ -61,4 +75,4 @@ public VersionedNamespace<IntVersion, UndertowSubsystemSchema> getNamespace() {
     public PersistentResourceXMLDescription getXMLDescription() {
         return UndertowPersistentResourceXMLDescriptionFactory.INSTANCE.apply(this);
```

Comment:
I am not sure if this is done already, but it would be good to add the new attributes to the XML description based on the stability level. You can check this using the `.since(versionNumber)` function. 
Something like this: https://github.com/wildfly/wildfly-core/blob/main/elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemSchema.java#L195-L197